doc/nrf/gs_modifying.rst

```diff
@@ -151,8 +221,39 @@ To locate a specific configuration option, use the filter (:guilabel:`Jump to` i
 The documentation for each :ref:`configuration option <configuration_options>` also lists the menu path where the option can be found.
 
 .. important::
-   All changes to the :file:`.config` file are lost when you clean your build directory.
-   You can save it to another location, but you must then manually copy it back to your build directory.
+   All changes to the :file:`.config` file are lost when you clean your build directory or you rebuild the project with the ``pristine`` target.
+   You can save the file to another location, but you must then manually copy it back to your build directory.
+   See Zephyr's :ref:`zephyr:application_rebuild` for more information.
+
+Changing the configuration permanently
+======================================
+
+To configure your application and maintain the configuration when you clean the build directory, you need to specify the configuration in one of the permanent configuration files.
+In most cases, this will be either the default :file:`prj.conf` file of the application or one of the configuration file overlays.
```

Comment:
I think writing:
`or a extra Kconfig fragment` is better than `or one of the configuration file overlays.`

do you have an opinion here @SebastianBoe .

Reply:
Implemented `or a extra Kconfig fragment`